src/components/sales/Order.vue

```diff
@@ -490,7 +491,7 @@
       </template>
 
       <template v-slot:body>
-        <OrderBill />
+        <OrderBill :print="print" :orderItems="orderItems" :cashReceived="cashReceived"/>
```

Comment:
Take Order object as prop.

Reply:
order object doesn't have batch object in order items instead it contains batch id. We need to show product name.